### PR TITLE
*: tiny cleanup.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@siddontang PTAL

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/vendor
+*/*.iml
+*/.idea
+*/deps.sh
+/syncer/syncer
```
Comment (siddontang) [inline .gitignore]: I suggest putting all binaries into root bin directory. or bin in every tool directory.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## tidb-tools
+
+tidb-tools are some useful tool collections for [TiDB](https://github.com/pingcap/tidb).
+
+[importer](https://github.com/pingcap/tidb-tools/tree/master/importer)
```
Comment (siddontang) [inline README.md]: you can use `(./importer)` directly.

**Comment by qiuyesuifeng** [comment]
@siddontang PTAL

File: Makefile [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
+importer:
+	mkdir -p bin && cd importer && $(GO) build && mv importer ../bin
 
-build: 
-	$(GO) build
-
-install: 
-	$(GO) install ./...
+syncer:
+	mkdir -p bin && cd syncer && $(GO) build && mv syncer ../bin
```
Comment (siddontang) [inline Makefile]: you can use `go build -o bin/syner syncer/main.go` directly

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/vendor
+/bin
+*/*.iml
+*/.idea
+deps.sh
+syncer.meta
+/importer/importer
```
Comment (siddontang) [inline .gitignore]: I think we can remove these now.

**Comment by siddontang** [comment]
we should clean up importer too.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**